syntaxes/npl-markdown-injection.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Generated with AI in a few minutes, though it seems to work nicely and the explanation made sense :D